.github/workflows/chart-release.yaml

```diff
@@ -85,9 +91,23 @@ jobs:
         run: |
           echo "chart_version=${{ inputs.chart_version }}" >> $GITHUB_ENV
 
+      - name: Determine Docker Tag
+        run: |
+          if [[ "${{ inputs.tag_generation }}" == 'ticket_from_branch' ]]; then
+          regex='^([[:alpha:]]+?-[[:digit:]]+)'
+          if [[ $GITHUB_REF_NAME =~ $regex ]]; then
+            DOCKER_TAG="${BASH_REMATCH[1]}"
+          fi
```

Comment:
If the regex doesn't match, DOCKER_TAG is not defined. It would be nice to have an error output in this case (e.g. `echo "::error:: Some explanation"` and then `exit 1`